### PR TITLE
Fe multi upload

### PR DESCRIPTION
**Pull request by khoathanglong**
- support multiple files upload
- remove upload button inside participants table
- require user to create a record before upload file
- allow to edit incomplete record (upload new files, delete uploaded files)
- allow to delete record

### CODE REVIEW

**Comment by nivemaham** [comment]
@khoathanglong Have you had a change to fix the test failures?

**Comment by khoathanglong** [comment]
> @khoathanglong Have you had a change to fix the test failures?

I have been involving in the meeting since yesterday. Now I'm working on it

File: radar-upload-frontend/.env.development [hunk 1]
```diff
@@ -1,5 +1,5 @@
-VUE_APP_BASE_URL=/upload/
-VUE_APP_API_BASE_URL=http://localhost:8085/upload/api
-VUE_APP_AUTH_API=http://localhost:8090/managementportal/oauth
-VUE_APP_AUTH_CALLBACK=http://localhost:8080/upload/login
+VUE_APP_BASE_URL=/
```
Comment (nivemaham) [inline radar-upload-frontend/.env.development]: Can you revert these changes to what we have on the original branch?

File: radar-upload-frontend/src/axios/__tests__/file.spec.js [hunk 2]
```diff
@@ -51,11 +51,15 @@ describe('axios/file', () => {
...
+    const onUploadProgress = 'progress';
+    const headers = { 'content-type': fileType };
     axios.put.mockResolvedValue();
-    return fileAPI.putRecords({ id, fileName, file })
+    return fileAPI.putRecords({
+      id, fileName, file, fileType, onUploadProgress,
+    })
       .then(() => {
-        expect(axios.put).toBeCalledWith(`records/${id}/contents/${fileName}`, file, { headers });
+        expect(axios.put).toBeCalled();
```
Comment (nivemaham) [inline radar-upload-frontend/src/axios/__tests__/file.spec.js]: Why did we remove the check with url and content?

Comment (khoathanglong) [reply radar-upload-frontend/src/axios/__tests__/file.spec.js]: I did mock the Onprogress function, but it seems Jest cannot compare if 2 functions are equal. Though it showed expected and actual value are the same, the test still failed

Comment (blootsvoets) [reply radar-upload-frontend/src/axios/__tests__/file.spec.js]: how about
```javascript
.toBeCalledWith(`records/${id}/contents/${fileName}`, file, expect.objectContaining({
     headers
}))
```

Comment (khoathanglong) [reply radar-upload-frontend/src/axios/__tests__/file.spec.js]: I did not know that expect can be inside another expect. But let me try :)

Comment (khoathanglong) [reply radar-upload-frontend/src/axios/__tests__/file.spec.js]: > how about
> 
> ```js
> .toBeCalledWith(`records/${id}/contents/${fileName}`, file, expect.objectContaining({
>      headers
> }))
> ```

it worked, thanks
